File: FreeYourFridge-SPA/src/app/_models/recipe.ts
import { Photo } from './photo';
import { MissedIngredients} from './missedIngredients';
import { UsedIngredients} from './usedIngredients';

export interface Recipe {
  id: number;
  title: string;
  photoUrl: string;
  instructions: string;
  summary: string;
  readyInMinutes: number;
  spoonacularScore: number;
  score: number;
 photos?: Photo[];
  missedIngredients: MissedIngredients[];
  usedIngredients: UsedIngredients[];
}

File: FreeYourFridge.API/Models/Recipe.cs
using System.Collections.Generic;

namespace FreeYourFridge.API.Models
{
    public class Recipe
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Image { get; set; }
        public string Summary { get; set; }
        public int ReadyInMinutes { get; set; }
        public double SpoonacularScore { get; set; }
        public string Instructions { get; set; }
        public Missedingredient[] MissedIngredients { get; set; }
        public Usedingredient[] UsedIngredients { get; set; }
    }

}